homework/fibonacci/fibonacci.hpp

```diff
@@ -1,11 +1,48 @@
 #pragma once
 
 int fibonacci_iterative(int sequence) {
-    // TODO: Your implementation goes here
+    
+    
+    int tab[sequence+1];
+    tab[0]=0;
+    tab[1]=1;
+    for (int i = 2; i<=sequence; i++)
+    {
+        tab[i]=tab[i-2]+tab[i-1];
+    }
+
+    sequence = tab[sequence];
+
+    if (sequence == 0)
+    {
+        return tab[0];
+    }
+    else if (sequence == 1)
+    {
+        return tab[1];
+    }
+    else 
+    {
+        return sequence;
+    }
+    
     return 0;
 }
 
 int fibonacci_recursive(int sequence) {
-    // TODO: Your implementation goes here
+
+    if (sequence ==0)
+    {
+        return 0;
+    }
+    else if(sequence == 1 || sequence ==2)
+    {
+        return 1;
+    }
+    else
```

Comment:
Sequence hold exactly the same value as you want to return for 0 and 1 :)

Better write:

`if (sequence < 2) {
    return sequence;
}`